src/Umbraco.Web/PublishedContentExtensions.cs

```diff
@@ -1746,6 +1746,11 @@ public static IEnumerable<IPublishedContent> Children(this IPublishedContent con
             return content.Children().Where(predicate);
         }
 
+        public static IEnumerable<IPublishedContent> Children(this IPublishedContent content, params string[] alias)

```

Comment:
Going to nitpick here @zpqrtbnk with documentation being generated automatically now from our code. Can we please add `<summary>` on what this is, even though it is obvious.

As it may be a bit confusing for people to know the difference between this extension method helper and just the Children property/collection